mindsdb/integrations/handlers/materialize_handler/materialize_handler.py

```diff
@@ -0,0 +1,55 @@
+from mindsdb.integrations.libs.const import HANDLER_CONNECTION_ARG_TYPE as ARG_TYPE
+from collections import OrderedDict
+
+
+from ..yugabyte_handler import Handler as YugaByteHandler
+
+
+class MaterializeHandler(YugaByteHandler):
```

Comment:
Let's extend the PostgreSQL handler that is most wildly used and better maintained